analyzers/tests/SonarAnalyzer.UnitTest/Helpers/MethodSignatureHelperTest.cs

```diff
@@ -363,17 +299,11 @@ private static InvocationContext CreateContextForMethod(string typeAndMethodName
         {
             var nameParts = typeAndMethodName.Split('.');
 
-            IEnumerable<(SyntaxNode node, string name)> invocation_identifierPairs = null;
-            if (snippet.IsCSharp())
-            {
-                invocation_identifierPairs = snippet.GetNodes<CSharpSyntax.InvocationExpressionSyntax>()
-                    .Select(n => ((SyntaxNode)n, n.Expression.GetIdentifier()?.Identifier.ValueText));
-            }
-            else
-            {
-                invocation_identifierPairs = snippet.GetNodes<VBSyntax.InvocationExpressionSyntax>()
+            var invocation_identifierPairs = snippet.IsCSharp()
```

Comment:
What do you think about extracting 2 local functions or methods and call them them here. Something similar with:
```
IEnumerable<(SyntaxNode node, string name)> GetCsharpNodes() =>
    snippet.GetNodes<CSharpSyntax.InvocationExpressionSyntax>().Select(n => ((SyntaxNode)n, n.Expression.GetIdentifier()?.Identifier.ValueText));

IEnumerable<(SyntaxNode node, string name)> GetVbNodes() =>
    snippet.GetNodes<VBSyntax.InvocationExpressionSyntax>().Select(n => ((SyntaxNode)n, n.Expression.GetIdentifier()?.Identifier.ValueText));
```

and then:
```
var identifierPairs = snippet.IsCSharp() ? GetCsharpNodes() : GetVbNodes();
```

I would also rename `invocation_identifierPairs` to `identifierPairs` or something similar.